weblab_cg/tests/test_chaste_cg_develop.py

```diff
@@ -162,9 +162,6 @@ def is_same(eq1, eq2):
         eq1 = sympy.sympify(eq1)
         eq2 = sympy.sympify(eq2)
 
-        if eq1.free_symbols != eq2.free_symbols:
```

Comment:
Why remove this?

Reply:
becuase it was telling me that 1.0 and 1.0 + 0.0*X wasn't the same which it is!